### ISSUE TITLE
The compiler does not support import { foo as bar } from '...'

### ISSUE THREAD

**Issue by jpzwarte** [ionic-team/stencil#2335]
**Stencil version:**

```
 @stencil/core@1.12.2
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**

**Expected behavior:**

**Steps to reproduce:**

When you rename an import using the `as` operator, the output in `components.d.ts` is invalid:

**Related code:**

```tsx
import { IconDefinition as FaIconDefinition } from '@fortawesome/fontawesome-svg-core';
```

Output (components.d.ts):
```ts
import { FaIconDefinition, } from "@fortawesome/fontawesome-svg-core";
```

**Comment by Nightmaster** [comment]
This references #2074, which did not had any answer from Stencil team for a long moment

**Comment by Nightmaster** [comment]
Duplicate of #2074, which did not had any answer from Stencil team for a long moment

**Comment by tanner-reits** [comment]
This was included in today's [v4.19.0 release](https://github.com/ionic-team/stencil/releases/tag/v4.19.0)!
